src/main/java/uk/gov/justice/hmpps/prison/api/model/BedAssignment.java

```diff
@@ -19,13 +23,33 @@
 @ToString
 @Data
 public class BedAssignment {
+    @ApiModelProperty(value = "Unique, numeric booking id.", position = 1, example = "1234134")
```

Comment:
Is this unique - thought it had to be combination of `bookingId` and `badAssignmentHistorySequence`?

Reply:
Good point, I'll update the docs :+1: